addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -159,6 +159,14 @@ const ColorPaletteWidget = Widget.extend({
     getColorNames: function () {
         return this.colorNames;
     },
+    /**
+     * Sets the currently selected color
+     *
+     * @param {string} color rgb[a]
+     */
+    setSelectedColor: function (color) {
+        this._selectColor({color: color}, 'set');
```

Comment:
Maybe do not trigger_up a "set" event for no reason. Just adapt the method so that the event_name parameter is optional and do not trigger_up anything unneeded.